js/docx/smoke/api_uni_color/to_json.js

```diff
@@ -1,6 +1,8 @@
 builder.CreateFile("docx");
     let oDocument = Api.GetDocument();
     let oParagraph = oDocument.GetElement(0);
+// Using an RGBColor instance is valid for testing UniColor.
```

Comment:
Maybe spare the inheritors from contacting the developers and write something like this

```js
// RGBColor is an inheritor of UniColor class.
// It is impossible to use an instance of UniColor class directly
```

Reply:
fix in new commit